rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -775,14 +882,21 @@ error_code cellGemGetState(u32 gem_num, u32 flag, u64 time_parameter, vm::ptr<Ce
 	}
 
 	if (g_cfg.io.move == move_handler::fake)
+		ds3_input_to_pad(gem_num, gem_state->pad.digitalbuttons, gem_state->pad.analog_T);
+	else if (g_cfg.io.move == move_handler::mouse)
 	{
-		map_to_ds3_input(gem_num, gem_state->pad.digitalbuttons, gem_state->pad.analog_T);
-		map_ext_to_ds3_input(gem_num, gem_state->ext);
+		mouse_input_to_pad(gem_num, gem_state->pad.digitalbuttons, gem_state->pad.analog_T);
+		mouse_pos_to_gem_state(gem_num, gem_state);
+	}
+
+	if (!(g_cfg.io.move == move_handler::null))
```

Comment:
Remove this check and always execute the code below.
Move the nulll handler check above the other move handler checks and return CELL_GEM_NOT_CONNECTED if move_handler::null

Reply:
move handler null already get gem not connected, i have initialized it in first
here is same, master already run code if have fake, i get just same